<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_01">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_02">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_03">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_06">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_04">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_05">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_07">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_08">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_09">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sendResult">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>380</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_00">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_point">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_less">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>290</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>﹣</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_divided">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>÷</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_times">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>200</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pluss">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>380</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>﹢</string>
     </property>
    </widget>
    <widget class="QFrame" name="result">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>751</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>721</width>
        <height>81</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>721</width>
        <height>81</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCALCULADORA">
    <property name="title">
     <string>CALCULADORA</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFECHAR">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>FECHAR</string>
    </property>
   </widget>
   <addaction name="menuCALCULADORA"/>
   <addaction name="menuFECHAR"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
